2-extra/3-groceries.js

```diff
@@ -52,6 +88,19 @@ let numberOfItemsPerWeek = {
   sunday: 0,
 };
 
+let arrayOfNum = []
+Object.values(weeklyMealPlan).forEach(val =>{
+  arrayOfNum.push(val.length)
+});
+numberOfItemsPerWeek.monday = arrayOfNum[0];
```

Comment:
In this case, you are going over all the elements of numberOfItemsPerWeek one by one to assign the arrayOfNum[incremental]. Is there another way to do this without having to add all of them one by one?